src/test/java/bio/terra/service/dataset/DatasetAzureIntegrationTest.java

```diff
@@ -454,19 +457,42 @@ public void datasetIngestFileHappyPath() throws Exception {
     TestUtils.verifyHttpAccess(snapshotParquetUrl, Map.of());
     verifySignedUrl(snapshotParquetUrl, steward(), "rl");
 
+    List<String> drsIds = new ArrayList<>();
     for (AccessInfoParquetModelTable table : snapshotParquetAccessInfo.getTables()) {
       if (tablesToCheck.contains(table.getName())) {
         String tableUrl = table.getUrl() + "?" + table.getSasToken();
         TestUtils.verifyHttpAccess(tableUrl, Map.of());
         verifySignedUrl(tableUrl, steward(), "rl");
+
+        // The vocabulary table has file data so test Drs on that one
+        // TODO: once we have an endpoint to expose parquet data, we should use that mechanism here
+        if (table.getName().equals("vocabulary")) {
+          List<Map<String, String>> records = ParquetUtils.readParquetRecords(tableUrl);
+          assertThat("2 rows are present", records.size(), equalTo(2));
+          // Extract the DRS Ids
+          drsIds.addAll(
+              records.stream()
+                  .map(r -> r.get("vocabulary_reference"))
+                  .collect(Collectors.toList()));
```

Comment:
Alternate form, a bit shorter
```suggestion
          records.stream()
              .map(r -> r.get("vocabulary_reference"))
              .forEach(drsIds::add);
```

Reply:
ooh, will do